Hazel.UnitTests/Dtls/DtlsConnectionTests.cs

```diff
@@ -313,6 +313,81 @@ public void TestMalformedConnectionData()
             }
         }
 
+
+        [TestMethod]
+        public void TestReorderedCertFragmentsConnects()
+        {
+            IPEndPoint captureEndPoint = new IPEndPoint(IPAddress.Loopback, 27511);
+            IPEndPoint listenerEndPoint = new IPEndPoint(IPAddress.Loopback, 27510);
+
+            bool serverConnected = false;
+            bool serverDisconnected = false;
+            bool clientDisconnected = false;
+
+            Semaphore signal = new Semaphore(0, int.MaxValue);
+
+            var logger = new TestLogger("Throttle");
+
+            using (SocketCapture capture = new SocketCapture(captureEndPoint, listenerEndPoint, logger))
+            using (DtlsConnectionListener listener = new DtlsConnectionListener(2, new IPEndPoint(IPAddress.Any, listenerEndPoint.Port), new TestLogger("Server")))
+            using (DtlsUnityConnection connection = new DtlsUnityConnection(new TestLogger("Client "), captureEndPoint))
```

Comment:
Noting the space at the end of `"Client "` but not in other TestClient names - on purpose?

Reply:
Naw, just copy paste stuff.